frontend/src/components/comparison-popup/comparison-popup.tsx

```diff
@@ -252,6 +252,9 @@ const ComparisonPopup: FC<ComparisonPopupProps> = ({
                       defaultValue="not_appliable"
                       disabled={!watch('model') || !watch('brand')}
                     >
+                      {availableComplectation.length === 0 && (
+                        <li className={styles.noOptions}>No options</li>
+                      )}
```

Comment:
I think ternary condition would be little more understandable here: 

`{availableComplectation.length ? (
                        availableComplectation.map((item) => (
                          <MenuItem key={item.id} value={item.id}>
                            {item.name}
                          </MenuItem>
                        ))
                      ) : (
                        <li className={styles.noOptions}>No options</li>
                      )}`

However your solution also not bad.